evaluation/swe_bench/scripts/_swe_lite_env_setup.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# THIS SCRIPT ONLY NEED TO BE RUN ONCE BEFORE EVALUATION
+set -e
+# throw error if user name is not opendevin
+if [ "$USER" != "opendevin" ]; then
+    echo "Error: This script is intended to be run by the 'opendevin' user only." >&2
+    exit 1
+fi
+
+# =======================================================
+# Install & Setup Conda
+
+# install if swe-util does NOT have conda
+if [ ! -d /swe_util/miniconda3 ]; then
+    pushd /swe_util
+    echo "Downloading and installing Miniconda3"
+    wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O Miniconda3-latest-Linux-x86_64.sh
+    bash Miniconda3-latest-Linux-x86_64.sh -b -p /swe_util/miniconda3
```

Comment:
Just a little question, could we run this without -b? The scripts from anaconda site include the license, which they display to the user, and ask for approval, unless run silently. The license is not open source, and while its terms are not surprising for educational institutions or individuals, they are significant for non-profit companies with more than 200 employees.

If we need to do this, maybe we can use a way to let users make an informed choice?

Reply:
Good point! How about `miniforge`? https://github.com/conda-forge/miniforge/blob/main/LICENSE Is BSD3 also gonna cause issues though?

Reply:
BSD3 is perfect! If we can do that instead, it's awesome ❤️